test/WebJobs.Script.Tests.Integration/ScriptHostEndToEnd/DirectLoadEndToEndTests.cs

```diff
@@ -91,7 +91,7 @@ public async Task Invoke_ExceptionThrown_DetailsLogged()
             var error = errorLogs[0];
             var invocationException = (FunctionInvocationException)error.Exception;
             Assert.Equal("Exception while executing function: Function1", invocationException.Message);
-            Assert.Equal("TestFunctions.Function1.Run", invocationException.MethodName);
+            Assert.Equal("TestFunctions.Functions.Run", invocationException.MethodName);
```

Comment:
With the test function updates, the checks in this test would fail without the fix